app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,231 @@
+body {
+  background-color: rgba(240, 245, 245, 0.5);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style-type: none;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.group_card {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: white;
+  margin: 24px;
+  border-radius: 24px;
+  padding: 12px;
+}
+
+.group_image {
+  height: 70px;
+  width: 70px;
+  border-radius: 12px;
+}
+
+.group_card_detail {
+  width: 70%;
+}
+
+.group_card_detail-inner {
+  display: flex;
+  justify-content: space-between;
+  height: 55px;
+  padding-top: 12px;
+}
+
+p.amount {
+  font-weight: bold;
+  padding: 8px;
+}
+
+.total_amount {
+  font-weight: bold;
+}
+
+h2.group_name a {
+  color: #434b54;
+  font-weight: bold;
+}
+
+.date {
+  color: #434b54;
+  font-size: 12px;
+}
+
+.button_box {
+  width: 100%;
+  height: 80px;
+}
+
+.push_box {
+  padding-top: 15px;
+}
+
+.add_category {
+  border: 2px solid #fff;
+  margin: 4px auto;
+  display: block;
+  width: 80%;
+  background-color: #5fb523;
+  padding: 12px 0;
+  border-radius: 2px;
+  color: #fff;
+}
+
+.total_box {
+  background-color: #f5f1f1;
+  padding: 24px 12px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.category_form {
+  border: 2px solid #fff;
+  width: 90%;
+  background-color: #fff;
+  margin: 12px auto;
+  display: block;
+  padding-bottom: 12px;
+  border-radius: 4px;
+}
+
+h3 {
+  font-weight: bold;
+  text-align: center;
+  padding: 24px 0;
+}
+
+.labela {
+  font-size: 18px;
+  font-weight: bold;
+  padding-left: 4px;
+  text-align: center;
+  display: block;
+}
+
+.form_divas {
+  display: block;
+  margin: 0 auto;
+  padding-left: 10%;
+  font-size: 18px;
+}
+
+.category_button {
+  padding: 8px;
+  margin: 12px auto;
+  display: block;
+  border: 2px solid #fff;
+  background-color: #5fb523;
+  border-radius: 4px;
+  color: #fff;
+}
+
+.chose-icon {
+  display: block;
+  margin: 0 auto;
+}
+
+.transact_card {
+  background: rgba(255, 255, 255, 0.8);
+  margin: 24px;
+  border-radius: 24px;
+  padding: 12px;
+}
+
+.login-field {
+  width: 80%;
+  height: 40px;
+  padding: 4px;
+  display: block;
+  margin: 12px auto;
+}
+
+.forgot_password {
+  text-align: center;
+  display: block;
+}
+
+.checkbox-field {
+  display: block;
+  margin: 0 10%;
+}
+
+.navbar {
+  background-color: #3778c2;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 12px;
+}
```

Comment:
- [ ] Kindly make sure the logout btn in the navbar is stilled as others button in the name ex: register.... this will make sure the design guidlines are followed

<img width="228" alt="image" src="https://github.com/chepkok3/Budget-app/assets/65068771/eddc7cbf-c501-4f15-ae51-cc2b320c2a83">